packages/webos/signLang/signLang.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable no-console */
+
+import {info} from '@enact/webos/pmloglib';
+import LS2Request from '../LS2Request';
+import {platform} from '../platform';
+
+let signLangEnabled = null;
+let appId = '';
+
+const checkSignLang = () => new Promise((resolve, reject) => {
+	if (signLangEnabled === null) {
+		new LS2Request().send({
+			service: 'luna://com.webos.settingsservice',
+			method: 'getSystemSettings',
+			subscribe: true,
+			parameters: {
+				'keys': ['signLanguageGuidance'],
+				'category': 'option'
+			},
+			onSuccess: function (res) {
+				info('enact_signLang_checkSignLang', {"onSuccess": res}, '');
+
+				if (res && res.settings.signLanguageGuidance === 'on') {
+					signLangEnabled = true;
+					appId = typeof window == 'object' && window.PalmSystem && window.PalmSystem.getIdentifier && window.PalmSystem.getIdentifier();
+					resolve();
+					return;
+				}
+
+				signLangEnabled = false;
+				reject();
+			},
+			onFailure: function (err) {
+				info('enact_signLang_checkSignLang', {"onFailure": err}, '');
+
+				reject('Failed to get sign language setting value: ' + JSON.stringify(err));
+			}
+		});
+	} else if (signLangEnabled) {
+		resolve();
+	} else {
+		reject();
+	}
+});
+
+const requestSignLang = (signLangId, focusOut, option) => () => new Promise((resolve, reject) => {
```

Comment:
This function should be renamed to..

```suggestion
const sendRequestSignLang = (signLangId, inactive, option) => () => new Promise((resolve, reject) => {
```